src/main/java/com/intuit/graphql/orchestrator/federation/EntityTypeMerger.java

```diff
@@ -61,6 +62,13 @@ private void pruneConflictingResolverInfo(EntityMergingContext entityMergingCont
       .filter(Objects::nonNull)
       .forEach(entityMetadata -> entityMetadata.getFields().remove(newEntityField.getName()));
     });
+
+    //If there are no remaining resolvers, then the resolver directive and ResolverArgument need to go
+    if(unifiedXtextGraph.getFieldResolverContexts().isEmpty() && currentFieldResolvers > 0) {
```

Comment:
currentFieldResolvers is the amount is was previously and the first check is what is now....I wanted to verify that the amount actually change and I could not set a flag in the removeIf, so went the route of confirming the size. I did not want that block of code to run every time the resolver context is 0 because that would break things too.

Reply:
makes sense,